2-dom-manipulation/form.js

```diff
@@ -0,0 +1,70 @@
+"use strict";
+
+const NAME_V = /^(([A-Za-z\s])+)$/;
+const EMAIL_V = /([a-z0-9]+[_az0-9\.-]*[a-z0-9]+)@([a-z0-9-]+(?:\.[a-z0-9-]+)*\.[a-z]{2,4})/;
+const PHONE_V = /[0-9]{2,}/;
+
+var htmlForm = document.querySelector('#form');
+htmlForm.addEventListener('submit', validateForm);
+var finalMessages = document.querySelector('#final-information');
+
+function validateForm(event) {
+    event.preventDefault();
+    let form = {
+        name: document.querySelector('#name'),
+        email: document.querySelector('#email'),
+        phone: document.querySelector('#phonen'),
+        courses: document.querySelectorAll('input[type="checkbox"]'),
+        message: document.querySelector('#texta')
+    };
+    printInHTML(form.name, form.email, form.phone, form.courses, form.message);
+}
```

Comment:
You can do something like this. The inputs will be available to other functions.
```
const htmlForm = document.querySelector('#form');
const courses = document.querySelectorAll('input[type="checkbox"]');
const name = document.querySelector('#name');
const email = document.querySelector('#email');
const phone =   document.querySelector('#phonen');
const message = document.querySelector('#texta');

let finalMessages = document.querySelector('#final-information');
htmlForm.addEventListener('submit', validateForm);

function validateForm(event) {
    event.preventDefault();
    let form = {
        name:    name,
        email:   email,
        phone:   phonen,
        courses: courses,
        message: message
    };
    printInHTML(form);
}
```